packages/scripts/src/upgrade-tests/register-new-contract/1_Propose.ts

```diff
@@ -0,0 +1,194 @@
+// This script generates and submits the transaction to register in the Registry and add to the Finder an arbitrary new
+// contract in mainnet and layer 2 blockchains. It can be run on a local hardhat node fork of the mainnet or can be run
+// directly on the mainnet to execute the upgrade transactions.
+// To run this on the localhost first fork mainnet into a local hardhat node by running:
+// HARDHAT_CHAIN_ID=1 yarn hardhat node --fork https://mainnet.infura.io/v3/<YOUR-INFURA-KEY> --port 9545 --no-deploy
+// Then execute the script:
+// NODE_URL_10=<OPTIMISM-NODE-URL> \
+// NODE_URL_288=<BOBA-NODE-URL> \
+// NODE_URL_137=<POLYGON-NODE-URL> \
+// NODE_URL_42161=<ARBITRUM-NODE-URL> \
+// yarn hardhat run ./src/upgrade-tests/register-new-contract/1_Propose.ts --network localhost
+
+const hre = require("hardhat");
+
+const { getAddress } = require("@uma/contracts-node");
+const { RegistryRolesEnum } = require("@uma/common");
+import { BigNumberish } from "@ethersproject/bignumber";
+import { BytesLike } from "@ethersproject/bytes";
+import { interfaceName } from "@uma/common";
+import {
+  ArbitrumParentMessenger,
+  Finder,
+  Governor,
+  GovernorHub,
+  GovernorRootTunnel,
+  Proposer,
+  Registry,
+} from "@uma/contracts-node/typechain/core/ethers";
```

Comment:
Can these be imported directly from @uma/contracts-node? Note: if we're not happy with the naming (ethers vs web3), we can change it!
```
import { ArbitrumParentMessengerEthers } from "@uma/contracts-node";
```

I think that's the syntax.

Reply:
Thank you. Fixed in latest commits